src/generators/clientContextFileGenerator.ts

```diff
@@ -269,6 +269,17 @@ function writeDefaultOptions(
     options = {};
   }
   ${defaults}
+
+  const userAgentPrefix =
+      options.userAgentOptions && options.userAgentOptions.userAgentPrefix
+        ? \`\${options.userAgentOptions.userAgentPrefix} ${
+          packageDetails.name
+        }/${packageDetails.version}\`
+        : \`${packageDetails.name}/${packageDetails.version}\`;
```

Comment:
same comment here about doing the concat work once.

Reply:
Done